pages/pil/index.js

```diff
@@ -5,20 +5,22 @@ module.exports = settings => {
   const app = Router({ mergeParams: true });
 
   app.param('pilId', (req, res, next, pilId) => {
-    if (pilId === 'create') {
-      return next('route');
-    }
-    return req.api(`/establishment/${req.establishmentId}/profiles/${req.profileId}/pil/${pilId}`)
-      .then(({ json: { data, meta } }) => {
-        req.model = data;
-        req.pil = data;
+    req.api(`/establishment/${req.establishmentId}/profile/${req.profileId}/pil/${pilId}`)
+      .then(response => {
         req.pilId = pilId;
-        req.model.openTasks = meta.openTasks || [];
+        req.pil = response.json.data;
+        req.pil.openTasks = response.json.meta.openTasks;
+        req.model = req.pil;
+        next();
       })
-      .then(() => next())
       .catch(next);
   });
 
+  app.use((req, res, next) => {
+    req.model = req.pil;
```

Comment:
This is already done at L13 above, how come it needs to be assigned again.

Reply:
it should be here, not above, as req.pil is set in the profile/id router - I've removed it from L13